mappings/extra/blockConnections.json

```diff
@@ -207,6 +207,32 @@
       "east": true
     }
   },
+  "grass_block[snowy=true]": {
```

Comment:
Don't add new entries, simply remove the properties part and brackets from the existing entry